csharp/src/Ice/StringUtil.cs

```diff
@@ -602,13 +602,64 @@ public static string UnescapeString(string s, int start, int end, string special
             return l.ToArray();
         }
 
+        public static string JoinString(string[] values, string delimiters)
```

Comment:
I think this method needs some documentation. I find it kinda confusing. Maybe we need 3 params?

Since this is really about properties should the name be something like `JoinStringProperty` or something?

Reply:
`JoinStringProperty` sounds fine, maybe I should just hardcode the delimiters used by GetPropertyAsList and have a single parameter `values`?

Reply:
I also think we could just hardcode the "property delimiters" `, \t\r\n`

Reply:
Yes, I think we can "hard code" all of this.